shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/repositories/RailRouteShapeRepository.kt

```diff
@@ -44,6 +48,50 @@ constructor(
     val response: MapFriendlyRouteResponse = MapFriendlyRouteResponse(emptyList()),
     val onGet: () -> Unit = {},
 ) : IRailRouteShapeRepository {
+
+    constructor(
+        objects: ObjectCollectionBuilder
+    ) : this(
```

Comment:
Needed to mock this in order to test that the route data was changing as expected